mac_pw_pool/setup.sh

```diff
@@ -74,10 +74,27 @@ grep -q homebrew /etc/paths || \
 # environment isn't loaded automatically.
 . /etc/profile
 
-msg "Installing podman-machine, testing, and CI deps. (~2m install time)"
+msg "Installing podman-machine, testing, and CI deps. (~5-10m install time)"
```

Comment:
Bump was based on observation w/ addition of 'krunkit' and breaking up the install command.  Brew produces quite a bit of output during each install, and install order is significant.  Like DNF repos., download times can also vary quite a bit due to many factors.

I chose to break up the installs in this PR mainly due to the [slp/homebrew-krunkit](https://github.com/slp/homebrew-krunkit/pull/1) problem I ran into.  This appears to have been resolved now, so I could switch to a package array w/ line comments and a single install command.

I'll give it one try, but if it fails this will need to be done as a followup.  The testing-feature is needed somewhat urgently and I won't be able to monkey around with the 3-hour DH spinup/spindown delays.

Reply:
FWIW, this is the setup.log from running the script in 783ae1e

[setup.log](https://github.com/user-attachments/files/16446046/setup.log)